bip-0085.mediawiki

```diff
@@ -438,8 +439,7 @@ BIP32, BIP39
 
 * 1.0 (2020-07)
 * 2.0.0 (2024-09-22)
-    * Swap chain code and private key bytes in application 32' for consistentcy with BIP-32 (major change)
-    * Correct derived entropy for application 128169' test vector (major change)
+    * Correct derived entropy for application 707764' test vector (bugfix)
```

Comment:
@akarve I assume this was bug in application number ? you mentioned `128169'` in previous [PR](https://github.com/bitcoin/bips/pull/1600) BUT wrong entropy was in `707764'`. I see no breaking changes in hex app

Reply:
The only vector that needs correction as of my simplified PR #1679 is [Hex (128169')](https://github.com/bitcoin/bips/pull/1679/files#diff-eae7a61b6d2a0b6091c1ce04618f216cbb871d987b985ba712de89e04014654cR264).

If somehow an extant implementation got the old test vector to work this correction would be a breaking change but I agree that if the correct BIP 85 algo is used then this is non-breaking :)